src/api/auth/loginApi.ts

```diff
@@ -0,0 +1,13 @@
+import HTTP, {BaseApi} from '../../modules/HTTPTransport';
+import {LoginRequest} from './types';
+import {ENDPOINTS} from '../consts';
+import {Success} from '../../shared/types';
+
+const authApiInstance = new HTTP(ENDPOINTS.AUTH.PATH);
+
+class LoginApi extends BaseApi {
+  public request(data:LoginRequest): Promise<Success> {
```

Comment:
Стоит привести к одному виду по всему проекту, для единообразия
```suggestion
  public request(data: LoginRequest): Promise<Success> {
```
```suggestion
  public request(data:LoginRequest):Promise<Success> {
```